### PR TITLE
ci: add useful workflows

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: .github/workflows/auto-remove-label.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  remove_label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.6.0
+      - uses: taiga-family/ci/actions/auto-remove-label@1.12.0
```
Comment (nsbarsukov) [inline .github/workflows/auto-remove-label.yml:11]: "state: need triage" label is not used inside Maskito.

I think that we can drop this workflow.

File: .github/workflows/auto-label.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+on: pull_request_review
+
+jobs:
+  label-when-approved:
+    name: Label when approved
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.6.0
+      - uses: taiga-family/ci/actions/variables@1.11.1
+      - uses: taiga-family/ci/actions/label-when-approved@1.11.1
```
Comment (nsbarsukov) [inline .github/workflows/auto-label.yml:11]: We need only one approve inside `Maskito` to merge PR.

Do we need this workflow ?

File: .github/workflows/auto-remove-label.yml (hunk 1 above)
Comment (splincode) [reply .github/workflows/auto-remove-label.yml:11]: Now it's used
https://github.com/taiga-family/maskito/labels

File: .github/workflows/auto-label.yml (hunk 2 above)
Comment (splincode) [reply .github/workflows/auto-label.yml:11]: fixed

**Pull request merged**